src/classes/MyHandler.ts

```diff
@@ -1636,12 +1636,10 @@ export = class MyHandler extends Handler {
         if (handledByUs && offer.data('switchedState') !== offer.state) {
             if (notify) {
                 if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
-                    this.bot.sendMessage(
-                        offer.partner,
-                        process.env.CUSTOM_SUCCESS_MESSAGE
-                            ? process.env.CUSTOM_SUCCESS_MESSAGE
-                            : '/pre ✅ Success! The offer went through successfully.'
-                    );
+                    this.bot.sendMessage(offer.partner, '/pre ✅ Success! The offer went through successfully.');
+                    if (process.env.CUSTOM_SUCCESS_MESSAGE) {
+                        this.bot.sendMessage(offer.partner, `${process.env.CUSTOM_SUCCESS_MESSAGE}`);
```

Comment:
no need to make it in template literals form. Just make it like:
```this.bot.sendMessage(offer.partner, process.env.CUSTOM_SUCCESS_MESSAGE);```

Reply:
Curly braces here can be removed for brevity if desired. Resultant code is shown below.

```
if (process.env.CUSTOM_SUCCESS_MESSAGE)
     this.bot.sendMessage(offer.partner, `${process.env.CUSTOM_SUCCESS_MESSAGE}`);
```

Reply:
no... it against ESLint and Prettier rules.